regression/test.pl

```diff
@@ -3,7 +3,7 @@
 use subs;
 use strict;
 use warnings;
-
+use Cwd;
```

Comment:
Nit picking: I wouldn't mind a blank line before and after this one (`Cwd` actually being a separate module, while the other `use` directives mainly affect behaviour).